eisbuk-admin/cypress/integration/calendar_spec.ts

```diff
@@ -0,0 +1,66 @@
+import { DateTime } from "luxon";
+import { PrivateRoutes, Routes } from "@/enums/routes";
+import { gus } from "@/__testData__/customers";
+import {
+  __currentDateId__,
+  __DayWithBookedSlots__,
+  __DayWithSlots__,
+} from "@/__testData__/testIds";
+
+describe("Date Switcher", () => {
+  describe("Badges", () => {
+    beforeEach(() => {
+      // our test data starts with this date so we're using it as reference point
+      const testDate = "2022-01-01";
+      const testDateLuxon = DateTime.fromISO(testDate);
+
+      cy.setClock(testDateLuxon.toMillis());
+      cy.initAdminApp().then((organization) =>
+        cy.updateFirestore(organization, [
+          "bookings.json",
+          "slots.json",
+          "customers.json",
+          "attendance.json",
```

Comment:
Attendance is probably unnecessary as it's created/updated through slot creation data trigger. That way you reduce possible inconsistencies in test data and at the same time test as close to production setting as it gets

Reply:
But we can go over this in detail in PP session